specs/wp-gutenberg-post-editor-spec.js

```diff
@@ -368,129 +367,130 @@ describe( `[${ host }] Gutenberg Editor: Posts (${ screenSize })`, function() {
 		} );
 	} );
 
-	xdescribe( 'Private Posts: @parallel', function() {
+	describe( 'Private Posts: @parallel', function() {
 		describe( 'Publish a Private Post', function() {
 			const blogPostTitle = dataHelper.randomPhrase();
 			const blogPostQuote =
 				'If you’re not prepared to be wrong; you’ll never come up with anything original.\n— Sir Ken Robinson\n';
 
+			before( async function() {
+				if ( driverManager.currentScreenSize() === 'mobile' ) {
+					await SlackNotifier.warn(
+						'Gutenberg private post spec currently not supported on mobile due to Gutenberg bug',
+						{ suppressDuplicateMessages: true }
+					);
+					return this.skip();
+				}
+			} );
+
 			step( 'Can log in', async function() {
-				let loginFlow = new LoginFlow( driver, 'gutenbergSimpleSiteUser' );
-				await loginFlow.loginAndStartNewPost();
+				this.loginFlow = new LoginFlow( driver, 'gutenbergSimpleSiteUser' );
+				return await this.loginFlow.loginAndStartNewPost( null, true );
 			} );
 
 			step( 'Can enter post title and content', async function() {
-				let editorPage = await EditorPage.Expect( driver );
-				await editorPage.enterTitle( blogPostTitle );
-				await editorPage.enterContent( blogPostQuote );
+				const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
+				await gEditorComponent.removeNUXNotice();
+				await gEditorComponent.enterTitle( blogPostTitle );
+				await gEditorComponent.enterText( blogPostQuote );
+				return await gEditorComponent.ensureSaved();
 			} );
 
 			step( 'Can disable sharing buttons', async function() {
-				let postEditorSidebarComponent = await PostEditorSidebarComponent.Expect( driver );
-				await postEditorSidebarComponent.expandSharingSection();
-				await postEditorSidebarComponent.setSharingButtons( false );
-				await postEditorSidebarComponent.closeSharingSection();
+				return await SlackNotifier.warn(
+					'Sharing buttons not currently available for Gutenberg in wp-admin'
+				);
+				//let postEditorSidebarComponent = await PostEditorSidebarComponent.Expect( driver );
+				//await postEditorSidebarComponent.expandSharingSection();
+				//await postEditorSidebarComponent.setSharingButtons( false );
+				//await postEditorSidebarComponent.closeSharingSection();
 			} );
 
 			step( 'Can allow comments', async function() {
-				let postEditorSidebarComponent = await PostEditorSidebarComponent.Expect( driver );
-				await postEditorSidebarComponent.expandMoreOptions();
-				await postEditorSidebarComponent.setCommentsForPost( true );
+				const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
+				await gEditorComponent.openSidebar();
+				const gEditorSidebarComponent = await GutenbergEditorSidebarComponent.Expect( driver );
+				await gEditorSidebarComponent.selectDocumentTab();
+				await driver.sleep( 3000 );
+				await gEditorSidebarComponent.collapseStatusAndVisibility(); // Status and visibility starts opened
```

Comment:
Do we need to collapse Status&Visibility? I think it'll work fine if it stays open.

Reply:
I think this is because we don't have a scroll into view and on mobile it may cause issues if it's open - might be safer to just close it